MapleServer2/PacketHandlers/Game/RequestCubeHandler.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Packets;
+using MapleServer2.Servers.Game;
+using Microsoft.Extensions.Logging;
+
+namespace MapleServer2.PacketHandlers.Game
+{
+    public class RequestCubeHandler : GamePacketHandler
+    {
+        public override RecvOp OpCode => RecvOp.REQUEST_CUBE;
+
+        public RequestCubeHandler(ILogger<RequestCubeHandler> logger) : base(logger) { }
+
+        public override void Handle(GameSession session, PacketReader packet)
+        {
+            byte mode = packet.ReadByte();
+
+            switch (mode)
+            {
+                case 0x11: // Pickup item
```

Comment:
Can this be converted to use enums? You can look at `ClubHandler.cs` for a good example of this, then you can remove the comments here